float32 z_delta
---
bool success